scripts/docker/DSS.Dockerfile

```diff
@@ -59,6 +59,7 @@ RUN set -eux \
     chmod 0777 /var/cache/yum && \
     mkdir /.cache && \
     chmod 0777 /.cache && \
+    python3 -m pip install gcovr \
```

Comment:
Let's lock it down to a version (6.0 - since I assume this is what you tested with). Also please add the `--no-cache-dir` and `--no-compile` options. You can probably add this after `ansible` on a new line (after line #65) which will make the resulting Docker image more compact.

Reply:
Made the listed changes